norwegian/terms.md

```diff
@@ -87,5 +107,11 @@ _Row_
 _signature_  
 Delen av en funksjon som beskriver [returtype](#returtype), navn og [argumenter](#argument)
 
+
+#### Transpilere
+_transpile_    
+Oversette, eller tranformere, kode fra ett programmeringsspråk til et annet. Kjente eksempler på dette er transpilering av 
+_TypeScript_ eller _CoffeScript_ til _JavaScript_. Må ikke misforstås med [Kompilering](Kompilere).
```

Comment:
Må ikke forveksles? Legg til VB til C# for fun? 😄

Reply:
hahah 😄

Poenget mitt er at det alt for mange steder på nettet står krevet varianter av "TypeScript is compiled to JavaScript". 

Dette er gjerne skrevet av personer som kanskje aldri faktisk _har_ brukt annet enn den type språk.  

Óg hver gang noen leser det, dør en kattunge i Kina, og jeg blir litt lei meg. 

MEN et eksempel eller to om andre transpilere er nok ikke dumt.

Reply:
Jeg tar i't. :P Merger.